examples/basic/version.tf

```diff
@@ -3,7 +3,7 @@ terraform {
   required_providers {
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = "= 1.56.1"
+      version = ">= 1.62.0"
```

Comment:
basic example should lock into lowest supported version if it can (so that would be 1.62.0)

Reply:
Done